src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentCost = 40 * days;
+  let discount = 0;
```

Comment:
[Checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md) 2nd item
```suggestion
  const discount = 0;
```